src/style.css

```diff
@@ -1 +1,74 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", "Montserrat", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background-color: white;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.logo-link {
+  display: flex;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  background-color: white;
```

Comment:
Delete it

Reply:
Agree, only with  background-color.

Reply:
Agree, only with  background-color.

Reply:
Yes, I mean only background-color: white;